conftest.py

```diff
@@ -463,8 +478,57 @@ def extra_checks_test_wrapper(selenium, trace_hiwire_refs, trace_pyproxies):
         assert delta_keys <= 0
 
 
+def _maybe_skip_test(item, delayed=False):
+    """If necessary skip test at the fixture level, to avoid
+
+    loading the selenium_standalone fixture which takes a long time.
+    """
+    skip_msg = None
+    # Testing a package. Skip the test if the package is not built.
+    match = re.match(
+        r".*/packages/(?P<name>[\w\-]+)/test_[\w\-]+\.py", str(item.parent.fspath)
+    )
+    if match:
+        package_name = match.group("name")
+        if not _package_is_built(package_name):
+            skip_msg = f"package '{package_name}' is not built."
+
+    # Common package import test. Skip it if the package is not built.
+    if str(item.fspath).endswith("test_packages_common.py") and item.name.startswith(
```

Comment:
Is possibly a test like this missing?
```suggestion
    if not skip_msg and str(item.fspath).endswith("test_packages_common.py") and item.name.startswith(
```

Reply:
Indeed it wouldn't hurt to check that.